services/worker-manager/src/estimator.js

```diff
@@ -6,23 +6,27 @@ class Estimator {
     this.monitor = monitor;
   }
 
-  async simple({workerPoolId, minCapacity, maxCapacity, workerInfo}) {
+  async simple({workerPoolId, minCapacity, maxCapacity, scalingRatio, workerInfo}) {
```

Comment:
maybe we just default this to `1.0` here?

Reply:
oops, I meant to add that default!